services/director-v2/requirements/_base.in

```diff
@@ -29,5 +30,5 @@ celery[redis]
 tenacity
 orjson
 
-# httpx [https://github.com/encode/httpx/releases/tag/0.15.0] due to PR#1285 [https://github.com/encode/httpx/pull/1285]
-httpx<0.15.0
```

Comment:
so if I get that correctly you account for that one using a global constraints.txt?

Reply:
yes